distributed/client.py

```diff
@@ -3494,8 +3494,8 @@ def get_scheduler_logs(self, n=None):
         Parameters
         ----------
         n : int
-            Number of logs to retrive.  Maxes out at 10000 by default,
-            confiruable in config.yaml::log-length
+            Number of logs to retrive.  Maxes out at 100000 by default,
+            confiruable in distributed.yaml::transition-log-length
```

Comment:
Both the scheduler and worker log length are configured using the [`"distributed.admin.log-length"`](https://github.com/dask/distributed/blob/25a2867e8e950d9cb51b071337ffc43ddaf0bb52/distributed/distributed.yaml#L176) configuration value (see [here](https://github.com/dask/distributed/blob/25a2867e8e950d9cb51b071337ffc43ddaf0bb52/distributed/node.py#L80-L97) for where this happens) 

```suggestion
            configurable with the ``distributed.admin.log-length``
            configuration value. 
```